docs/tutorials/random.rst

```diff
@@ -15,69 +14,68 @@ name across the display::
-an item chosen at random. This item (the randomly chosen name) is the argument
-for ``display.scroll``.
+Lista (``names``) zawiera siedem imion zdefiniowanych jako łańcuchy znaków.
+Ostatnia linia jest *zagnieżdżona* (efekt "onion" zaprezentowany wcześniej):
+metoda ``random.choice`` przyjmuje jako argument listę ``names`` oraz zwraca
+jej losowy element. Element ten (losowo wybrane imię) jest argumentem dla
+``display.scroll``.
 
-Can you modify the list to include your own set of names?
+Czy możesz zmodyfikować listę tak, by zawierała wybrane przez ciebie imiona?
 
-Random Numbers
+Losowe Liczby
 ++++++++++++++
 
-Random numbers are very useful. They're common in games. Why else do we have
-dice?
+Losowe liczby są bardzo użyteczne. Często używane są w grach. Po cóż innego
+mamy kostki?
 
-MicroPython comes with several useful random number methods. Here's how to
-make a simple dice::
+MicroPython zawiera wiele użytecznych metod powiązanych z losowymi liczbami.
+Oto przykład prostej kości do gry::
 
     from microbit import *
     import random
 
     display.show(str(random.randint(1, 6)))
 
-Every time the device is reset it displays a number between 1 and 6. You're
-starting to get familiar with *nesting*, so it's important to note that
-``random.randint`` returns a whole number between the two arguments, inclusive
-(a whole number is also called an integer - hence the name of the method).
-Notice that because ``display.show`` expects a character then we use the
-``str`` function to turn the numeric value into a character (we turn, for
-example, ``6`` into ``"6"``).
-
-If you know you'll always want a number between ``0`` and ``N`` then use the
-``random.randrange`` method. If you give it a single argument it'll return
-random integers up to, but not including, the value of the argument ``N``
-(this is different to the behaviour of ``random.randint``).
-
-Sometimes you need numbers with a decimal point in them. These are called
-*floating point* numbers and it's possible to generate such a number with the
-``random.random`` method. This only returns values between ``0.0`` and ``1.0``
-inclusive. If you need larger random floating point numbers add the results
-of ``random.randrange`` and ``random.random`` like this::
+Po każdym restarcie urządzenia wyświetla ono numer z zakresu 1-6. Zaczynasz
+zapoznawać się z *zagnieżdżaniem*. Warto więc tylko zauważyć, że ``random.randint``
+zwraca liczbę całkowitą z przedziału domkniętego pomiędzy dwoma argumentami
+(liczba całkowita nazywana jest integer, stąd nazwa metody). Zauważ, że skoro
+``display.show`` oczekuje znaku alfabetycznego, używamy funkcji ``str`` do zamiany
+numeru na znak (na przykład: ``6`` w ``"6"``).
+
+W wypadku, gdy zawsze potrzebujesz liczny z przedziału pomiędzy ``0`` a ``N``,
+używaj metody ``random.randrange``. Gdy podasz jej pojedynczy argument, zwróci
+losową liczbę całkowitą z przedziału zamkniętego od ``0`` do ``N`` (w
```

Comment:
Co to jest "przedział zamknięty"? Może chodzi o "domknięty"? Patrz też powyższa uwaga o "włącznie".